soh/soh/Enhancements/cosmetics/CosmeticsEditor.cpp

```diff
@@ -92,11 +99,11 @@ typedef struct {
     bool advancedOption;
 } CosmeticOption;
 
-#define COSMETIC_OPTION(id, label, group, defaultColor, supportsAlpha, supportsRainbow, advancedOption) \
-    { id, { \
+#define COSMETIC_OPTION(id, label, group, defaultColor, supportsAlpha, supportsRainbow, advancedOption)                                    \
+    { id, {                                                                                                                                \
         CVAR_COSMETIC(id ".Value"), CVAR_COSMETIC(id ".Rainbow"), CVAR_COSMETIC(id ".Locked"), CVAR_COSMETIC(id ".Changed"), label, group, \
-        defaultColor, defaultColor, \
-        supportsAlpha, supportsRainbow, advancedOption \
+        defaultColor, defaultColor,                                                                                                        \
+        supportsAlpha, supportsRainbow, advancedOption                                                                                     \
```

Comment:
What happened with all the whitespace here?

Reply:
I lined all the slashes up, it's easier to see when turning on unified diff in github.